src/lsp-server.ts

```diff
@@ -69,14 +69,17 @@ export class LspServer {
         this.logger.log('documentSymbol', params);
         const filePath = fileURLToPath(params.textDocument.uri);
         const rootPath = this.findBelongedRootPath(filePath);
+        if (!rootPath) {
+            return [];
+        }
         const relativePath = path.relative(rootPath, filePath);
         const stream = ctags.createReadStream(path.resolve(rootPath, this.tagFileName));
         return new Promise<SymbolInformation[]>(resolve => {
             let results: SymbolInformation[] = [];
             stream.on('data', (tags) => {
                 const definitions = tags.filter(tag => tag.file === relativePath);
                 for (let def of definitions) {
-                    let symbolInformation = SymbolInformation.create(def.name, SymbolKind.Array,
+                    let symbolInformation = SymbolInformation.create(def.name, SymbolKind.Method,
```

Comment:
Unrelated to this PR, why we use `Method` kind to create symbol information. There are so many programming languages ctag support, different languages have differnt top-most declaration kinds. However, we can discuss this in the future.